Add about release content for SI (#3870)
wso2/docs-apim: en/docs/get-started/about-this-release.md

Streaming Integrator serves the streaming-based integrations and acts as the event stream provider for Streaming APIs in the data plane of the API Manager platform. This is a newly added component into WSO2 API Manager 4.0.0.

> It is now available to download from [here](https://github.com/wso2/streaming-integrator/releases/download/v4.0.0/wso2si-4.0.0.zip).

Reviewer: General comment:

Better to describe acronyms like CDC, EOF...

Author: Done

### New features

> - **[Deploy AsyncAPI definitions to the WSO2 API-M Services]({{base_path}}/use-cases/streaming-usecase/exposing-stream-as-managed-api-in-service-catalog/)**

Reviewer: ```suggestion
- **[Deploy AsyncAPI definitions to WSO2 API-M Services]({{base_path}}/use-cases/streaming-usecase/exposing-stream-as-managed-api-in-service-catalog/)**
```

> 	 WSO2 Streaming Integrator has the inbuilt capability to directly deploy siddhi applications with Async Api definition to WSO2 Api Manager services which afterwards can be used to deploy an api.

Reviewer: ```suggestion
	 WSO2 Streaming Integrator has the inbuilt capability to directly deploy siddhi applications with Async API definitions to WSO2 API Manager services. These services can then be used to create an API and deploy in the API Manager runtime.
```

> - **SSE Sink / Source support over http**

Reviewer: ```suggestion
- **SSE Sink / Source support over HTTP**
```

> 	 SSE is an HTTP-based protocol that allows one-way communication similar to WebHooks, from the server to the client. For more information , see [SSE Server Sink](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#sse-server-sink) , [SSE Source](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#sse-source).

Reviewer: ```suggestion
	 SSE is an HTTP-based protocol that allows one-way communication (similar to WebHooks) from the server to the client. For more information, see [SSE Server Sink](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#sse-server-sink) and [SSE Source](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#sse-source).
```

> - **WebSub Sink / Source support over http**

Reviewer: ```suggestion
- **WebSub Sink / Source support over HTTP**
```

> 	 Implementation is based on the [WebSub](https://www.w3.org/TR/websub/) protocol specification.WebSubs only allows one-way communication, from a caller web app to a callee web app. For more information , see [WebSub Sink](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-sink) , [WebSub Source](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-source).

Reviewer: what implementation? A word seems to be missing at the start

"<The SOMETHING> implementation is based on the [WebSub](https://www.w3.org/TR/websub/) protocol specification. WebSubs only allows one-way communication from a caller web app to a callee web app. For more information, see [WebSub Sink](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-sink) and [WebSub Source](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-source)."

Author: Hmm need to convey that our websub Sink / Source is done according to mentioned protocol specification.

> 	 Above Source and Sink implementation are based on the [WebSub](https://www.w3.org/TR/websub/) protocol specification.WebSubs only allows one-way communication, from a caller web app to a callee web app. For more information , see [WebSub Sink](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-sink) , [WebSub Source](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-source).

Reviewer: ```suggestion
	 Source and Sink implementations are based on the [WebSub](https://www.w3.org/TR/websub/) protocol specification.WebSubs only allows one-way communication, from a caller web app to a callee web app. For more information, see [WebSub Sink](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-sink) and [WebSub Source](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-source).
```

- **Improved MongoDB Store**
    

> 	 Several improvements are done like allow encryption protocol to be configurable, Object_ID support when joining and other bug fixes. For more information , see [Mongo DB Store](https://siddhi-io.github.io/siddhi-store-mongodb/api/2.1.2/#mongodb-store).

Reviewer: 'Object_ID-support when joining' what?
Consider changing it to something like - 'The Object_ID can be used to .... when .. joins ....'

"The improvements include the following: Configurable encryption protocol, Object_ID-support when joining, and other bug fixes. For more information, see [Mongo DB Store](https://siddhi-io.github.io/siddhi-store-mongodb/api/2.1.2/#mongodb-store)."

Author: improved

> 	 Several improvements are done like allow encryption protocol to be configurable, Object_ID support when searching specific values in the store and other bug fixes. For more information , see [Mongo DB Store](https://siddhi-io.github.io/siddhi-store-mongodb/api/2.1.2/#mongodb-store).

Reviewer: ```suggestion
	 Several improvements are done, such as making the encryption protocol configurable, Object_ID support when searching specific values in the store, and other bug fixes. For more information, see [Mongo DB Store](https://siddhi-io.github.io/siddhi-store-mongodb/api/2.1.2/#mongodb-store).
```

	 
- **Improved IO CDC Source**
    

> 	 CDC can be used to capture insert / update / delete actions from databases. Release includes improvement where with single CDC configuration it can handle all three actions. For more information , see [CDC Source](https://siddhi-io.github.io/siddhi-io-cdc/api/2.0.10/).

Reviewer: ```suggestion
	 CDC can be used to capture insert /update/delete actions from databases. With the new improvements, you can handle all three actions using a single CDC configuration. For more information, see [CDC Source](https://siddhi-io.github.io/siddhi-io-cdc/api/2.0.10/).
```

	 
- **Synchronous publishing support for IO Kafka**
    

> 	 IO Kafka allows seamless connection with kafka broker and retrieves the events into the streaming integrator. Release includes synchronous publishing in a more controlled manner and other bug fixes. For more information , see [IO-Kafka](https://github.com/siddhi-io/siddhi-io-kafka).

Reviewer: 'synchronous publishing' of what?

" IO Kafka allows the Streaming Integrator to seamlessly connect with the Kafka broker and get events. The new improvements include synchronous publishing in a more controlled manner and other bug fixes. For more information, see [IO-Kafka](https://github.com/siddhi-io/siddhi-io-kafka)."

Author: added some more context

> 	 IO Kafka allows seamless connection with kafka broker and retrieves the events into the streaming integrator. Release includes synchronous publishing of events which allows more controlled event publishing and other bug fixes. For more information , see [IO-Kafka](https://github.com/siddhi-io/siddhi-io-kafka).

Reviewer: ```suggestion
	 IO Kafka allows seamless connection with the Kafka broker and retrieves the events into the Streaming Integrator. This release includes synchronous publishing of events, which allows more controlled event publishing and other bug fixes. For more information, see [IO-Kafka](https://github.com/siddhi-io/siddhi-io-kafka).
```

	 
- **Improved JSON Mapper to support null values**
    

> 	 JSON mapper provides capability to retrieve or publish events in JSON format. For more information , see [Map-JSON](https://github.com/siddhi-io/siddhi-map-json).

Reviewer: The description doesn't say anything about 'null values' support.

" The JSON mapper allows the Streaming Integrator to get events and publish events in JSON format. For more information , see [Map-JSON](https://github.com/siddhi-io/siddhi-map-json)."

Author: improved

	 
- **Improved KeyValue Mapper to support force casting**
    

> 	 KeyValue mapper provides capability to retrieve or publish events in as key value format. For more information , see [Map-KeyValue](https://siddhi-io.github.io/siddhi-map-keyvalue/api/2.1.1/).

Reviewer: ```suggestion
	 The KeyValue mapper allows the Streaming Integrator to get events and publish events as a key-value pair. For more information, see [Map-KeyValue](https://siddhi-io.github.io/siddhi-map-keyvalue/api/2.1.1/).
```

	 
- **Improved IO File Sink and Source**
    

> 	 IO File sink / source can be used to read or write into a file. With the new versions it has the support for EOF in other modes, support for file trailers, file writing mechanisms and SFTP support. For more information , see [IO-File](https://siddhi-io.github.io/siddhi-io-file/api/2.0.16/).

Reviewer: ```suggestion
	 IO File sink/source is used to read or write into a file. With the new improvements, the Streaming Integrator supports EOF in other modes, file trailers, file-writing mechanisms, and SFTP. For more information, see [IO-File](https://siddhi-io.github.io/siddhi-io-file/api/2.0.16/).
```

	 
- **Improved Store RDBMS support**
    

> 	 Store RDBMS can be used to connect with any rdbms type. With the new versions it has the support to add null values to RDBMS tables and improved metrics support. For more information , see [Store RDBMS](https://github.com/siddhi-io/siddhi-store-rdbms).

Reviewer: Better not refer to Store RDBMS as an object in a sentence...
Is this correct?
"The **Store RDBMS** extension is used to connect with any RDBMS type. The new improvements include the possibility to add null values to RDBMS tables and enhanced metrics."

Author: This is a technical word for one of the extensions.  I feel this is fine because user should know that this is technical word for said extension

Reviewer: okay.. Just added some minor suggestions.. See commit suggestion below.

> - **Added Kafka and HTTP metrics monitoring capabilities**

Reviewer: ```suggestion
- **Kafka and HTTP metrics monitoring**
```

> 	 Store RDBMS can be used to connect with any rdbms type. With the new versions it has the support to add null values to RDBMS tables and improved metrics support. For more information , see [Store RDBMS](https://github.com/siddhi-io/siddhi-store-rdbms).

Reviewer: ```suggestion
	 Store RDBMS can be used to connect with any RDBMS type. With the new versions, it supports adding null values to RDBMS tables and includes improved metrics support. For more information, see [Store RDBMS](https://github.com/siddhi-io/siddhi-store-rdbms).
```

> 	 This release provides Kafka and HTTP metrics monitoring capability. In the previous release, this was only available for File, RDBMS and CDC. The metrics can be published to a pre-configured [Prometheus](https://prometheus.io/) server and then you can either create your own [Grafana](https://grafana.com/) dashboard or use the ETL dashboard which is already created by us. For more information on how to configure metrics, refer to [Monitoring ETL Flows](https://apim.docs.wso2.com/en/4.0.0/observe/si-observe/viewing-dashboards/) documentation page.

Reviewer: The description says "previously, these features were available for certain scenarios..". So, what's new now? Improved capabilities?
Suggestion:
"This Streaming Integrator release provides improved Kafka and HTTP metrics monitoring capabilities. Previously, Kafka and HTTP metrics monitoring capabilities were only available for File, RDBMS, and CDC. You can now publish the metrics to a pre-configured [Prometheus](https://prometheus.io/) server and then you can either create your own [Grafana](https://grafana.com/) dashboard or use the ETL dashboard which is already available with the Streaming Integrator. For more information on how to configure metrics, see [Monitoring ETL Flows](https://apim.docs.wso2.com/en/4.0.0/observe/si-observe/viewing-dashboards/) documentation page."

Author: No suggestion need to change. as per the topic what newly added was Kafka and HTTP metrics monitoring capabilities. Earlier we had the monitoring capabilities only for  File, RDBMS, and CDC. So I guess this is fine. WDYT ?
